File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/bitfinex/BitfinexPubTickerResponseBean.java
package com.androidevlinux.percy.UTXO.data.models.bitfinex;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by percy on 18/11/17.
 */

public class BitfinexPubTickerResponseBean implements Serializable
{

    @SerializedName("mid")
    @Expose
    private String mid;
    @SerializedName("bid")
    @Expose
    private String bid;
    @SerializedName("ask")
    @Expose
    private String ask;
    @SerializedName("last_price")
    @Expose
    private String lastPrice;
    @SerializedName("low")
    @Expose
    private String low;
    @SerializedName("high")
    @Expose
    private String high;
    @SerializedName("volume")
    @Expose
    private String volume;
    @SerializedName("timestamp")
    @Expose
    private String timestamp;
    private final static long serialVersionUID = -5611742423241891237L;

    public String getMid() {
        return mid;
    }

    public void setMid(String mid) {
        this.mid = mid;
    }

    public String getBid() {
        return bid;
    }

    public void setBid(String bid) {
        this.bid = bid;
    }

    public String getAsk() {
        return ask;
    }

    public void setAsk(String ask) {
        this.ask = ask;
    }

    public String getLastPrice() {
        return lastPrice;
    }

    public void setLastPrice(String lastPrice) {
        this.lastPrice = lastPrice;
    }

    public String getLow() {
        return low;
    }

    public void setLow(String low) {
        this.low = low;
    }

    public String getHigh() {
        return high;
    }

    public void setHigh(String high) {
        this.high = high;
    }

    public String getVolume() {
        return volume;
    }

    public void setVolume(String volume) {
        this.volume = volume;
    }

    public String getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(String timestamp) {
        this.timestamp = timestamp;
    }

}

File: app/src/main/java/com/androidevlinux/percy/UTXO/data/models/bitfinex/BitfinexPubTickerResponseBean.kt
package com.androidevlinux.percy.UTXO.data.models.bitfinex

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

/**
 * Created by percy on 18/11/17.
 */

class BitfinexPubTickerResponseBean : Serializable {

    @SerializedName("mid")
    @Expose
    var mid: String? = null
    @SerializedName("bid")
    @Expose
    var bid: String? = null
    @SerializedName("ask")
    @Expose
    var ask: String? = null
    @SerializedName("last_price")
    @Expose
    var lastPrice: String? = null
    @SerializedName("low")
    @Expose
    var low: String? = null
    @SerializedName("high")
    @Expose
    var high: String? = null
    @SerializedName("volume")
    @Expose
    var volume: String? = null
    @SerializedName("timestamp")
    @Expose
    var timestamp: String? = null

    companion object {
        private const val serialVersionUID = -5611742423241891237L
    }

}
